libs/beacon/src/dkg_output.cpp

```diff
@@ -22,7 +22,6 @@ using fetch::beacon::DkgOutput;
 
 DkgOutput::DkgOutput()
 {
-  fetch::crypto::mcl::details::MCLInitialiser();
```

Comment:
Since we call this in constellation main it is probably safe not to include these @jinmannwong